stunnel/dune

```diff
@@ -4,12 +4,13 @@
   (wrapped false)
   (flags (:standard -w -37))
   (libraries astring
-             forkexec
-             threads
              fmt
-             ptime
+             forkexec
+             ipaddr
              logs.fmt
+             ptime
              safe-resources
+             threads
```

Comment:
`threads` is not needed. The one that is usually is needed is `threads.unix`. You can confirm that by adding `(implicit_transitive_deps false)` to `dune-project`

Reply:
The libraries `uuidm` and `xapi-inventory` should added, the code depends on them, but they are current being implicitely being imported through other dependencies.

Reply:
I'll make these changes - I'd rather do this on master however, and then we can merge them into feature/REQ-403 later